node/Cargo.toml

```diff
@@ -63,6 +61,7 @@ pretty_assertions = "1.4.0"
 quick-protobuf = "0.8.1"
 quote = "1.0.33"
 rand = "0.8.0"
+rand04 = { package = "rand", version = "0.4" }
```

Comment:
since you contribute to the fork, can you upgrade rand as well?

Reply:
If you mean to upgrade `pairing_ce`, I didn't attempt to do so because the underlying `ff_ce` is also using "0.4".

Reply:
ack